src/imago.js

```diff
@@ -2,11 +2,11 @@
   'use strict';
 
   if (typeof define === 'function' && define.amd)
-    define('imago-js', [], new Imago());
+    define('imago-js', [], Imago);
   else if (typeof exports !== 'undefined')
-    exports.Imago = new Imago();
+    exports.Imago = Imago;
   else
-    window.Imago = new Imago();
+    window.Imago = Imago;
```

Comment:
You need to stay with instantiation in agnostic case: `window.Imago = new Imago();`. Because imago module wrapper is passed by argument, if you don`t execute/instantiate it will return the module wrapper instead Imago. That's why the tests are not passing. Tests use Imago agnosticly, without any module loader/pattern (like AMD/CommonJS/ES6).